### ISSUE TITLE
Plugin doesn't work on window.location.hostname

### ISSUE THREAD

**Issue by v-tank** [amilajack/eslint-plugin-compat#376]
Environment: VSCode
Plugin version: 3.7.0
Browserslist default: "> 0.5%, last 2 versions, Firefox ESR, not dead"
Issue: I am using some properties that are a part of window.location (such as hostname) but the plugin did not catch that it's incompatible with Opera 67; it only catches 'location.hostname' and not 'window.location.hostname'. Is this expected behavior? How can I make sure that explicit properties from the window object are not being missed by the linter?
Thank you.

**Comment by mnicpt** [comment]
Same here. No support for Opera 23+ but shows supported on MDN.

**Comment by wulfy** [comment]
Same here

the current line code 
``` export const currentRootDomain = getRootDomain(window.location.hostname); ```

failed during tests with this elsint error : 

```37:48  error  location.hostname() is not supported in Opera 69  compat/compat ``` 

This error is surprising because this property exists for a long time.
I didn't manage to find a documentation about hostname property on Opera 69, but I downloaded Opera 69 (from here http://get.opera.com/ftp/pub/opera/desktop/ ) and tested the result : 

<img width="893" alt="Capture d’écran 2020-09-25 à 20 15 40" src="https://user-images.githubusercontent.com/724239/94302079-ef530380-ff6b-11ea-8f45-5ef6639b44b0.png">

As you can see, Opera 69 supports hostname property

**Comment by afdev82** [comment]
Any news about this issue?
Thank you!
